src/shared/utils/alias_to_year_mapper.ts

```diff
@@ -3,10 +3,16 @@ import { AliasName } from "../alias-mappings";
 export function mapYearToAlias(courseYear: number): AliasName {
 	const userYear = 2000 + courseYear;
 	const year = new Date().getFullYear() - (new Date().getMonth() < 7 ? 1 : 0);
-	const currentYear = year - userYear;
 
-	return (
-		[AliasName.YEAR1, AliasName.YEAR2, AliasName.YEAR3][currentYear] ??
-		AliasName.CS_MASTER
-	);
+	const current = year - userYear;
+
+	switch (current) {
+		case 0:
+			return AliasName.YEAR1;
+		case 1:
+			return AliasName.YEAR2;
+		case 2:
+			return AliasName.YEAR3;
+	}
+	return AliasName.CS_MASTER;
 }
```

Comment:
Just interested, why was this changed and reverted?

Reply:
I'm not really a fan of switch cases and switched to an array-indexed approach but I got some feedback from Herman that it might have made the code more difficult to read so I reverted it back